### ISSUE TITLE
Bounding box issues. 

### ISSUE THREAD

**Issue by slumberingWoods** [Zank0ku/The-Pirate-s-Adventures#3]
The png size of the pirate needs to be reduced. When the transparent areas of the pirate touch the goblin, it still counts the empty part of the image as the pirate. The fix seems to be removing the transparent areas using an image editing tool and adjusting the canvas size.

**Comment by Zank0ku** [comment]
Yeah, we tried out different sizes and this one worked best. Thanks.
